Bug: build issue wrong path

### Describe the bug

In all PRs the build is being broken due to wrong path based on my understanding. 

here is the issue we got: `npm ERR! path /home/runner/work/ai/ai/package.json`

related to #143 and #141 and #101

### Steps to reproduce

1. open a PR 
2. you will see the issue 
3. here is an example action: https://github.com/open-sauced/ai/actions/runs/5098790076/jobs/9166141097?pr=143

### Affected services

browser-extension

### Platforms

_No response_

### Browsers

_No response_

### Environment

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

--- a/.github/workflows/development.yml
+++ b/.github/workflows/development.yml
@@ -14,5 +14,19 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps: 
-        - name: "📦 run build"
-          run: npm run build
+      - name: "☁️ checkout repository"
+        uses: actions/checkout@v2
+
+      - name: "🔧 setup node"
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: 18
+
+      - name: "🔧 install npm@latest"
+        run: npm i -g npm@latest
+
+      - name: "📦 install dependencies"
+        uses: bahmutov/npm-install@v1
+
+      - name: "📦 run build"
+        run: npm run build

